src/app/courses/courses.component.html

```diff
@@ -0,0 +1,21 @@
+<app-header>
+  <p class="header-name">Dzmitry</p>
+  <app-button class="header-button" [buttonText]="'Logout'"></app-button>
+
+</app-header>
```

Comment:
App header should be displayed on all feature pages which are defined in the point one of the task, so it is needed to move it in app component.